src/parser/TraverseSentences.h

```diff
@@ -462,6 +462,28 @@ class FetchEdgesSentence final : public Sentence {
     std::unique_ptr<EdgeKeyRef>     keyRef_;
     std::unique_ptr<YieldClause>    yieldClause_;
 };
+
+class LimitSentence final : public Sentence {
+public:
+    explicit LimitSentence(int64_t skip, int64_t count) : skip_(skip), count_(count) {
+        kind_ = Kind::kLimit;
+    }
+
+     std::string toString() const override;
+
+     int64_t skip() {
+        return skip_;
+    }
+
+     int64_t count() {
+        return count_;
+    }
+
+ private:
+    int64_t    skip_{-1};
```

Comment:
you alse define skip_ in LimiteExecutor with initial number 0. why not the same.